### PR TITLE
Add node package using grpc dynamic codegen

### PR DESCRIPTION
**Pull request by djfarrelly**
### Purpose
This will allow us to publish this easily to npm

Enables the new gRPC metric tracking in https://github.com/bufferapp/buffer-js-buffermetrics/pull/4

### Review
Hey @michael-erasmus and @davidgasquez, this is the minimal code I ended up with after adjusting the import paths and created a main `index.js` file to export all the protobuf/grpc code dynamically. Let me know what you think!

### CODE REVIEW

File: buda/events_collector_service.proto [hunk 1]
```diff
@@ -2,8 +2,8 @@ syntax = "proto3";
 
 package buda;
 
-import "buda/funnel.proto";
-import "buda/funnel_event.proto";
+import "funnel.proto";
```
Comment (djfarrelly) [inline buda/events_collector_service.proto]: The paths for the js code have to be relative

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (michael-erasmus) [inline package.json]: Hey @djfarrelly! Curious if this needs to live in the project root or could perhaps be moved to the `package/node` directory?

**Comment by davidgasquez** [comment]
Hey @djfarrelly! Just checking, do we still need the relative imports in the proto files or is it ok to close this PR?

**Comment by michael-erasmus** [comment]
Hey @davidgasquez! I think at the moment we do still need the relative imports for javascript. 

I have planned at looking into it today as I work on the new visits pipeline though. Will let you know @davidgasquez!

**Comment by davidgasquez** [comment]
Hey @michael-erasmus! Since we're now using `budapest` I think we can close this PR. Feel free to reopen if I'm missing something. 😅

**Comment by michael-erasmus** [comment]
@davidgasquez agreed! thanks for cleaning it up.

**Pull request closed without merging**